build/stage-build.yml

```diff
@@ -72,13 +75,14 @@ jobs:
     pool:
       vmImage : $(windowsHostedAgentImage)
     variables:
-      ArtifactName: $(TestsArtifactName)_${{ parameters.applicationEnvironment }}
+      ArtifactName: ${{ parameters.artifactName }}
```

Comment:
Make sure you are using it though.

Reply:
Am I not ?

Reply:
Don,t know, did not see the change. Might just be me, confirm please.

Reply:
it is